aerospike-graph/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: "2.0.0"
+version: "2.2.0"
```

Comment:
We need to publish this new helm-chart version. So, add an entry for this in the `docs/index.html` via command
`helm repo index .`

Note: Only keep changes for your graph helm-chart entry in index.html and revert other changes done by helm index command. You can refer this PR: https://github.com/aerospike/helm-charts/pull/3/files